team13-fydp/fydp
# Java translation w/ even distribution + 80 mins art/ music

Removing the comments for the constraints associated with even distribution as well as upping the number of minutes for art and music from 40 mins to 80 mins.

**Tests:**
1. Run this branch locally on your own device and ensure the model runs 
2. Ensure all constraints are met

_Reference Objective Value: 24200_
